bin/postman-test/DataDumps/logging/populate.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
@weichou1229 Thanks for adding this but see my other comments regarding how it's a different methodology from the existing test data population @cloudxxx8

Reply:
@tsconn23 I had changed dump file to json format , please check .